test/testcafe/spec/ChallengeCustomAppPush_spec.js

```diff
@@ -500,6 +506,16 @@ test
       .contains('Haven\'t received a push notification yet? Try opening Custom Push on your phone.');
   });
 
+test.skip
```

Comment:
test is added but skipped?

Reply:
I marked it with `skip` because test takes 30s to complete

Reply:
Expensive, but the test isn't useful unless it's run. Please enable

Reply:
or try test timer in jest so you can mock it